zh/upgrade-tidb-operator.md

```diff
@@ -17,12 +17,23 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
 
 1. 更新 Kubernetes 的 CustomResourceDefinition (CRD)。关于 CRD 的更多信息，请参阅 [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)。
 
-    {{< copyable "shell-regular" >}}
+    * 如果 Kubernetes 版本大于等于 1.16:
 
-    ```shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
-    kubectl get crd tidbclusters.pingcap.com
-    ```
+         {{< copyable "shell-regular" >}}
+
+         ```shell
+         kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
+         kubectl get crd tidbclusters.pingcap.com
+         ```
+    
+     * 如果 Kubernetes 版本小于 1.16:
+
+         {{< copyable "shell-regular" >}}
+
+         ```shell
+         kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd_v1beta1.yaml && \
```

Comment:
Is this case verified?

Reply:
not yet, I will have a try

Reply:
verified, it works for kubernetes v1.17 or less, but v1beta1 CRDs is incompatible with v1.18 or later